### PR TITLE
feat: toggle room visibility

### LINKED ISSUE #67: Feature Request : Toggle between public and private rooms
An option that enables rooms to toggle between private and public visibility even after they are created.

### PR DESCRIPTION
**Pull request by giuseppecampanelli**
Quick way to allow room privacy toggling which fixes #67 . Underlying code to switch between public & private can be cleaned up but I can only get to it later. If you need this out now, it has a first iteration.

### CODE REVIEW

File: kofta/src/vscode-webview/Routes.tsx [hunk 1]
```diff
@@ -50,7 +50,11 @@ export const Routes: React.FC<RoutesProps> = () => {
         setCurrentRoom((cr) =>
           !cr || cr.id !== roomId ? cr : { ...cr, name, isPrivate }
         );
-        toast("room is now public", { type: "info" });
+        if (isPrivate) {
+          toast("room is now public", { type: "info" });
+        } else {
+          toast("room is now private", { type: "info" });
+        }
```
Comment (alii) [inline kofta/src/vscode-webview/Routes.tsx:57]: You could convert this to a single line

```ts
toast(`Room is now ${isPrivate ? "private" : "public"}`, { type: "info" });
```

**Review by alii — CHANGES_REQUESTED** [review]

**Comment by benawad** [comment]
oops didn't mean to change status, but looks good to me, let me know when you want merge

File: kofta/src/vscode-webview/Routes.tsx (hunk 1 above)
Comment (giuseppecampanelli) [reply kofta/src/vscode-webview/Routes.tsx]: Changed!

**Comment by giuseppecampanelli** [comment]
@benawad ok to merge

**Pull request merged**